drivers/include/periph/adc.h

```diff
@@ -128,6 +128,20 @@ int adc_init(adc_t line);
  */
 int32_t adc_sample(adc_t line, adc_res_t res);
 
+/**
+ * @brief   Capture multiple ADC samples from one or multiple ADC lines
+ *
+ * @param[in]  lines        line(s) to sample
+ * @param[in]  lines_numof  number of ADC lines to sample
+ * @param[out] bufs         target buffer(s)
+ * @param[in]  buf_len      number of samples to sample per line
```

Comment:
I think this needs to be reworded. I do not understand how many buffers I need and how to specify their size.

Reply:
I re-shuffled the arguments to make the compiler able to check the array sizes, I hope this also makes this easier to understand.